game_eggs/steamcmd_servers/ark_survival_ascended/egg-ark--survival-ascended.json

```diff
@@ -0,0 +1,164 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2023-10-29T12:19:41-04:00",
+    "name": "ARK: Survival Ascended",
+    "author": "blood@darkartsgaming.com",
+    "description": "ARK is reimagined from the ground-up into the next-generation of video game technology with Unreal Engine 5! Form a tribe, tame & breed hundreds of unique dinosaurs and primeval creatures, explore, craft, build, and fight your way to the top of the food-chain. Your new world awaits!",
+    "features": [
+        "steam_disk_space"
+    ],
+    "docker_images": {
+        "ghcr.io\/parkervcp\/steamcmd:proton": "ghcr.io\/parkervcp\/steamcmd:proton"
+    },
+    "file_denylist": [],
+    "startup": "rmv() { echo \"stopping server\"; rcon -t rcon -a 127.0.0.1:${RCON_PORT} -p ${ARK_ADMIN_PASSWORD} SaveWorld && rcon -t rcon -a 127.0.0.1:${RCON_PORT} -p ${ARK_ADMIN_PASSWORD} DoExit && wait ${ARK_PID}; echo \"Server Closed\"; exit; }; trap rmv 15 2; proton run .\/ShooterGame\/Binaries\/Win64\/ArkAscendedServer.exe \"{{SERVER_MAP}}?listen?MaxPlayers={{MAX_PLAYERS}}?SessionName={{SESSION_NAME}}?Port={{SERVER_PORT}}?QueryPort={{QUERY_PORT}}?RCONPort={{RCON_PORT}}?RCONEnabled=True?ServerAdminPassword={{ARK_ADMIN_PASSWORD}}$( [  \"$SERVER_PVE\" == \"0\" ] || printf %s '?ServerPVE=True' )?ServerPassword={{SERVER_PASSWORD}}{{ARGS_PARAMS}}\" -WinLiveMaxPlayers={{MAX_PLAYERS}} -oldconsole -servergamelog$( [ \"$BATTLE_EYE\" == \"1\" ] || printf %s ' -NoBattlEye' ) {{ARGS_FLAGS}} & ARK_PID=$! ; tail -c0 -F .\/ShooterGame\/Saved\/Logs\/ShooterGame.log --pid=$ARK_PID & until echo \"waiting for rcon connection...\"; (rcon -t rcon -a 127.0.0.1:${RCON_PORT} -p ${ARK_ADMIN_PASSWORD})<&0 & wait $!; do sleep 5; done",
```

Comment:
`?ServerAdminPassword=\"{{ARK_ADMIN_PASSWORD}}\"`
I just tried on a fresh install of the latest changes it it still grabs the ServerPassword 
![image](https://github.com/parkervcp/eggs/assets/34715358/c83c7d64-7dcd-4f5f-a229-84ff9145a2b4)

Reply:
Yeah i have the same lmao

Reply:
fresh install != fresh re-install

Please make sure you test by deleting the entire server and re-creating it.

Reply:
Yes that's what I mean by a fresh install. I donwloaded the latest egg and imported it under a new nest to make sure I don't mix them up. Then I created a new server which had the problem.

Reply:
https://github.com/parkervcp/eggs/pull/2529/commits/eacb24563c4b14c1389dbc5762ad8418d9e544f6
Please let me know if latest commit fixes the issues. Also, please make sure you update your existing server(s) with the latest Startup Command from the egg.